packages/mask/src/components/CompositionDialog/useSubmit.ts

```diff
@@ -75,3 +65,34 @@ async function pasteImage(
         reason,
     })
 }
+
+// TODO: Provide API to plugin to postprocess post content,
+// then we can move these -PreText's and meta readrs into plugin's own context
+function decorateEncryptedText(encrypted: string, t: I18NFunction, meta?: Meta) {
+    const hasOfficalAccount = isTwitter(activatedSocialNetworkUI) || isFacebook(activatedSocialNetworkUI)
+    const officalAccount = isTwitter(activatedSocialNetworkUI) ? t('twitter_account') : t('facebook_account')
```

Comment:
Maybe move these two lines into a function?

Reply:
Need to pass down `t`, that will be complicated.